tests/cypress/support/e2e.ts

```diff
@@ -20,7 +20,7 @@ declare global {
   namespace Cypress {
     interface Chainable {
       // Functions declared in commands.ts
-      open3dotsMenu(name: string, selection?: string): Chainable<Element>;
+      open3dotsMenu(name: string, selection?: string, checkMenu?: boolean): Chainable<Element>;
```

Comment:
if accepted previous change on paremeter name, change `checkMenu`  to `checkNotInMenu`